browser/brave_content_browser_client.cc

```diff
@@ -1033,12 +1033,8 @@ BraveContentBrowserClient::CreateThrottlesForNavigation(
   return throttles;
 }
 
-bool BraveContentBrowserClient::OverrideWebPreferencesAfterNavigation(
-    WebContents* web_contents,
-    WebPreferences* prefs) {
-  bool changed =
-      ChromeContentBrowserClient::OverrideWebPreferencesAfterNavigation(
-          web_contents, prefs);
+bool PreventDarkModeFingerprinting(WebContents* web_contents,
```

Comment:
I didn't notice where you put this, it should go at the top of the file in an anonymous namespace